Type-and-reword (#39)
devcontainers/feature-starter: README.md

Features are meant to be easily sharable units of dev container configuration and installation code.  

> This repo contains a **GitHub Action** [workflow](.github/workflows/release.yaml) that will publish each feature to GHCR. 

Reviewer: ```suggestion
This repo contains a **GitHub Action** [workflow](.github/workflows/release.yaml) that will publish each Feature to GHCR. 
```

> *Allow GitHub Actions to create and approve pull requests* should be enabled in the repo's `Settings > Actions > General > Workflow permissions` for auto generation of `src/<feature>/README.md` per feature (merging in existing `src/<feature>/NOTES.md`).

Reviewer: ```suggestion
*Allow GitHub Actions to create and approve pull requests* should be enabled in the repository's `Settings > Actions > General > Workflow permissions` for auto generation of `src/<feature>/README.md` per feature (which merges any existing `src/<feature>/NOTES.md`).
```

Features are meant to be easily sharable units of dev container configuration and installation code.  

This repo contains a **GitHub Action** [workflow](.github/workflows/release.yaml) that will publish each Feature to GHCR. 

> *Allow GitHub Actions to create and approve pull requests* should be enabled in the repository's `Settings > Actions > General > Workflow permissions` for auto generation of `src/<feature>/README.md` per feature (which merges any existing `src/<feature>/NOTES.md`).

Reviewer: ```suggestion
*Allow GitHub Actions to create and approve pull requests* should be enabled in the repository's `Settings > Actions > General > Workflow permissions` for auto generation of `src/<feature>/README.md` per Feature (which merges any existing `src/<feature>/NOTES.md`).
```
